### PR TITLE
Break main.css into multiple files

### PR DESCRIPTION
**Pull request by roblarsen**
This PR breaks main.css into multiple files, adds a build steps that concats them for regular distribution and also copies the the component files to `dist` so that they can be ingested individually into a project. 

(forgot the WIP label)

- [x] keep LICENSE.txt and remove the extensionless LICENSE
- [x] there's a jshint config added that should probably be removed if we're using eslint instead
- [x] exclude package-lock.json from being included (add it to the .gitignore file)
- [x] I think it makes sense to add autoprefixer to the build process... but we could do this later in a new PR.
- [x] We need to expand the README - important thing is to mention browser support.(which would make sense to match latest HTML5Boilerplate)
- [x] remove main.css from root

### CODE REVIEW

**Comment by roblarsen** [comment]
@coliff The build warning is just a warning. Apparently it's fine.

**Comment by roblarsen** [comment]
👷

**Comment by Jeroen-Matthijssens** [comment]
@coliff Maybe this is the wrong place to ask, but I've seen it suggested in a couple of places to remove the `package-lock.json` file from revision control. However on the npm website suggests to do the opposite (https://docs.npmjs.com/files/package-lock.json). 

So I'm wondering what the motivation is for removing `package-lock.json`?

ps: I'm not saying you are wrong, just that I don't know the arguments in favour of removing. I'm not that familiar with nodejs build tools and practices.

**Comment by roblarsen** [comment]
@Jeroen-Matthijssens There are some dependency trees where weird things happen([a story about package-lock.json in two commits](https://github.com/jquery/jquery/search?q=package-lock.json&type=Commits).  In this particular case, the only people who will run `npm install` inside the folder are those looking to develop the underlying code (me, for now.) Because of that package-lock.json is just noise.  

@coliff I'm not sure what time zone you're in, but whenever you get this take another look at the state of this branch. I'd like to merge it soon.

File: README.md [hunk 1]
```diff
@@ -1,9 +1,141 @@
...
+Also, they should always be included last, so that the other styles can be
+overwritten.
+
+
+## Browser support
+
+* Chrome *(latest 2)*
+* Edge *(latest 2)*
+* Firefox *(latest 2)*
+* Internet Explorer 9+
```
Comment (coliff) [inline README.md]: should be IE11

File: src/_print.css [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+   https://www.phpied.com/delay-loading-your-print-css/
+   ========================================================================== */
+
+   @media print {
+    *,
+    *:before,
+    *:after {
+        background: transparent !important;
+        color: #000 !important; /* Black prints faster */
+        -webkit-box-shadow: none !important;
```
Comment (coliff) [inline src/_print.css]: remove this to leave it for Autoprefixer to deal with?

File: README.md (hunk 1 above)
Comment (roblarsen) [reply README.md]: 👍

File: src/_print.css (hunk 2 above)
Comment (roblarsen) [reply src/_print.css]: 👍

File: README.md [hunk 3]
```diff
@@ -1,9 +1,141 @@
...
+### _print.css
+
+Lastly, we provide some useful print styles that will optimize the printing
+process, as well as make the printed pages easier to read.
+
+At printing time, these styles will:
+
+* strip all background colors, change the font color to black, and remove the
+  `text-shadow` — done in order to [help save printer ink and speed up the
+  printing process](http://www.sanbeiji.com/archives/953)
```
Comment (coliff) [inline README.md]: http://www.sanbeiji.com/archives/953 is a dead link - lets remove it

**Comment by coliff** [comment]
looking good though the formatting/indenting of the CSS is a bit inconsistent and not matching the `.editorconfig`. Some attributes have tabs set to 4 spaces.

**Pull request merged**